src/pages/ViewInterviewPage.tsx

```diff
@@ -225,36 +253,36 @@ const QuestionCard = ({
 }: {
   question: TechnicalQuestion | BehavioralQuestion;
   index: number;
-  type: 'technical' | 'behavioral';
+  type: "technical" | "behavioral";
```

Comment:
Let's use the enum here. `QuestionType.technical | QuestionType.behavioral`

Reply:
resolved in ej pr